src/RecoObjects/Jet.cpp

```diff
@@ -259,7 +259,7 @@ const ParticlePointer Jet::smear_jet(const ParticlePointer jet, const ParticlePo
 	double energy_smeared = ptScale * energy_unsmeared;
 	double px_smeared = ptScale * px_unsmeared;
 	double py_smeared = ptScale * py_unsmeared;
-	double pz_smeared = ptScale * pz_unsmeared;
+	double pz_smeared = pz_unsmeared;
 
```

Comment:
The recommendation is to scale the p<sub>T</sub> only. No mention of p<sub>Z</sub>.